bindings/nostr-sdk-js/src/client/mod.rs

```diff
@@ -38,8 +38,8 @@ impl JsClient {
 
     /// Get current `Keys`
     #[wasm_bindgen(getter)]
-    pub fn keys(&self) -> JsKeys {
-        self.inner.keys().into()
+    pub async fn keys(&self) -> JsKeys {
```

Comment:
Isn't turning a synchronous function into an asynchronous one in the public API a breaking change?

Reply:
Yes, it's a breaking change. I usually try to avoid breaking change, but in the last versions happened more frequently for some methods to implement new features.

In the README.md I written that, since the library it's in ALPHA state, the APIs **could/will change in breaking ways**.